src/cli/commands/files/cat.js

```diff
@@ -23,10 +23,12 @@ module.exports = {
           if (err) {
             throw err
           }
-          console.log(res.toString())
+
+          res.pipe(process.stdout)
         })
         return
       }
+
       ipfs.files.cat(path, (err, file) => {
         if (err) {
           throw (err)
```

Comment:
This can be simplified now, since it doesn't matter anymore if the daemon is on, js-ipfs-api also has `ipfs.files.cat` and it is the same API